### DISCUSSION TITLE
Channel q values

### DISCUSSION THREAD

**Discussion by vmunro2** [vdemichev/DiaNN#718] [Q&A] [answered]
Hi Vadim,

When searching DIA-NN with channels, should you still filter on Q.Value as well as Channel.Q.Value, or is this no longer relevant?

Thanks!
Victoria

**Comment by vdemichev** [comment, marked as answer]
Hi Victoria,

Q.Value filtering is automatic. Channel q-value - if you feel you need confidence that you really see the peptide in the specific channel. For it, values in the range 0.01 - 0.5 are reasonable.

Best,
Vadim

**Reply by vmunro2** [reply]
Thank you very much!
